lib/galaxy/config.py

```diff
@@ -276,6 +276,11 @@ def __init__( self, **kwargs ):
         # On can mildly speed up Galaxy startup time by disabling index of help,
         # not needed on production systems but useful if running many functional tests.
         self.index_tool_help = string_as_bool( kwargs.get( "index_tool_help", True ) )
+        self.tool_name_boost = kwargs.get( "tool_name_boost", None )
```

Comment:
It is a pretty common convention that the example configuration documents the default parameters. Otherwise you need to look at the code to know what the default is. Users expect that uncommenting a line in that config without changing the value will not change anything.